src/toplevel/learnocaml_toplevel_output.ml

```diff
@@ -180,6 +180,36 @@ let output_html ?phrase output html =
   Js_utils.Manip.appendChild output.container div ;
   insert output ?phrase (Html (html, div)) div
 
+let get_fresh_id =
+  let r = ref 0 in
+  fun () -> incr r ; !r
+
```

Comment:
This piece of code should be moved to a general-purpose utility module.

Reply:
As said in this review https://github.com/ocaml-sf/learn-ocaml/pull/352#discussion_r462463243, I'm not sure it is supposed to be moved in a new module as should only be used by this module.

Reply:
@yurug ?